src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -84,6 +84,7 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
             for (Client sampleClient : getSampleClients(sampleAb)) {
                 sampleAb.addClient(sampleClient);
             }
+            sampleAb.setClientCounter("6");
```

Comment:
Maybe instead of hard-coding the number out, we can use the length of the sampleClients array
```suggestion
            int numOfClients = getSampleClients(sampleAb).length;
            sampleAb.setClientCounter(valueOf(numOfClients)); // indexing starts from 0
```